pkgs/top-level/python-packages.nix

```diff
@@ -1653,7 +1653,12 @@ self: super: with self; {
 
   catalogue = callPackage ../development/python-modules/catalogue { };
 
-  catboost = callPackage ../development/python-modules/catboost { };
+  catboost = callPackage ../development/python-modules/catboost {
+    catboost = pkgs.catboost.override {
+      enablePython = true;
+      python3 = python;
+    };
+  };
```

Comment:
This is rather odd, can't we move the python build completely into pkgs.catboost?

Reply:
We could perhaps build python packages in one derivation, but it would increase complexity and be inefficient with long builds each time, even if we only change the python package.

I switched to python3Packages as input instead of python3, as many other packages do.